style/edit-job-definitions.css

```diff
@@ -0,0 +1,8 @@
+.input-file-snapshot.draghover {
+  background: rgba(33, 150, 243, 0.1);
```

Comment:
Could we use a theme variable for this, as on the following line?

Reply:
We could but there is no existing variable that matches this color and this is the color used in `dockpanel` dragndrop overlay implementation in jupyterlab (overlay that is active on drag over all parts of our page except of `Input file snapshot` field and that opens a dragged Notebook in a split view on drop). And we need to match that color. 

![Screen Shot 2022-11-17 at 10 21 03 AM](https://user-images.githubusercontent.com/26686070/202526840-1cc873af-d3a6-4b16-842e-31d5a20c684d.png)